ecommerce/api.py

```diff
@@ -162,3 +166,40 @@ def make_reference_id(order):
             A reference number for use with CyberSource to keep track of orders
     """
     return "MM-{}-{}".format(settings.CYBERSOURCE_REFERENCE_PREFIX, order.id)
+
+
+def get_new_order_by_reference_number(reference_number):
```

Comment:
Yes, it's referring to a newly created order, one with a `status` of `CREATED`. What would you rename it to?